9.2/Dockerfile.rhel7

```diff
@@ -44,7 +41,14 @@ RUN yum install -y yum-utils gettext && \
     localedef -f UTF-8 -i en_US en_US.UTF-8 && \
     mkdir -p /var/lib/pgsql/data && chown postgres.postgres /var/lib/pgsql/data && \
     test "$(id postgres)" = "uid=26(postgres) gid=26(postgres) groups=26(postgres)" && \
-    chmod -R go+rwx /var/lib/pgsql && chmod go+w /var/run/postgresql
+    # Loosen permission bits to avoid problems running container with arbitrary UID
```

Comment:
oops, nm, was looking at the output from the jenkins test job...was running both in parallel.